test/integration/scenarios/network/peer_disconnect.js

```diff
@@ -123,8 +112,8 @@ module.exports = params => {
 				// Need to keep one peer so that we can validate
 				// Duplicate socket connection exists or not
 				it('stop all the nodes in the network except node_0', done => {
-					for (let i = 1; i < totalPeers; i++) {
-						stopNode(`node_${i}`);
+					for (let i = 1; i < networkParameters.TOTAL_PEERS; i++) {
+						common.stopNode(`node_${i}`);
 					}
 					setTimeout(() => {
```

Comment:
Same as above

Reply:
Stop node calls synchronous `childProcess.execSync`, so it's synchronous as well. Later on we are waiting 2000 ms to ensure the process finished though. The proper solution would have to change childProcess.execSync to childProcess.exec and utilize the output. 
It can be added as a separate issue as the next following improvements.